docs/src/main/paradox/full-example.md

```diff
@@ -2,6 +2,6 @@
  
 Take a look at the definition of Actors, their companion objects, and Messages:
  
-@@snip [AkkaQuickstart.scala]($g8src$/scala/$package$/AkkaQuickstart.scala)
+@@snip [AkkaQuickstart.scala](/src/main/g8/src/main/scala/$package$/AkkaQuickstart.scala)
```

Comment:
Where do you see this as broken?

In the published guide it looks right: https://developer.lightbend.com/guides/akka-quickstart-scala/full-example.html

Reply:
@patriknw 
<img width="636" alt="2018-11-29 16 12 03" src="https://user-images.githubusercontent.com/9194205/49205195-9064e780-f3f1-11e8-8a8b-0a915d6eb959.png">

https://github.com/akka/akka-quickstart-scala.g8/blob/2.5.x/docs/src/main/paradox/full-example.md

Although it seems that the link of [AkkaQuickstart.scala] returns 404, is it right?
(Am I misunderstanding?)